tests/test-setup.js

```diff
@@ -0,0 +1,3 @@
+const dotenv = require("dotenv");
```

Comment:
It would have been really good to include some info about how to set up the database and what the `env` values are. Without them its not possible to run the tests or to know what the requirements are
(This would live in the readme, not this file. Just mentioning it here as the readme isnt included in the PR through no fault of your own)